projects/distributed-tracing/src/shared/dashboard/interaction/detail-sheet/container/detail-sheet-interaction-container.component.ts

```diff
@@ -0,0 +1,12 @@
+import { ChangeDetectionStrategy, Component, Inject, InjectionToken } from '@angular/core';
+
+@Component({
+  selector: 'htc-detail-sheet-interaction-container',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `<ng-container [hdaDashboardModel]="this.detailModel"></ng-container>`
+})
+export class DetailSheetInteractionContainerComponent {
+  public constructor(@Inject(DETAIL_SHEET_INTERACTION_MODEL) public readonly detailModel: object) {}
+}
+
+export const DETAIL_SHEET_INTERACTION_MODEL = new InjectionToken<unknown>('DETAIL_SHEET_INTERACTION_MODEL');
```

Comment:
object, right? should match the type on line 9

Reply:
okay